### DISCUSSION TITLE
[Question]: Frigate without home assistant on my personnal web application

### DISCUSSION THREAD

**Discussion by tyrelire** [blakeblackshear/frigate#10416] [Ask A Question]
### What is your question:

it's possible to use frigate without home assistant just on go2rtc because I'm making a website under Symfony 7 for my surveillance cameras and I'm wondering if I could integrate it into my code

**Comment by NickM-27** [comment]
yes, frigate does not require home assistant

**Reply by tyrelire** [reply]
It would be possible to have documentation or videos that explain how to use it (a sort of wiki) because I am new to this area and I don't really know where to find the information yet.

**Reply by NickM-27** [reply]
https://docs.frigate.video/guides/getting_started
